tests/openwisp2/settings.py

```diff
@@ -52,7 +52,9 @@
     'leaflet',
     # rest framework
     'rest_framework',
+    'rest_framework.authtoken',
     'rest_framework_gis',
+    'drf_yasg',
```

Comment:
great :+1:, please add a mention in the README about how to see the API docs (add a screenshot please, it will help ... and in the future we'll be adding a lot more screenshots to the READMEs of each module).